trustfall_core/src/graphql_query/error.rs

```diff
@@ -1,3 +1,4 @@
+//! Errors from parsing GraphQL
```

Comment:
```suggestion
//! Errors from parsing Trustfall queries
```
Because there are also errors that can come up when parsing Trustfall schemas too.